### PR TITLE
Dec 2019 changelog

### PR DESCRIPTION
**Pull request by kevingranade**
#### Summary
SUMMARY: None
lol

#### Purpose of change
Update the changelog, as usual I automatically imported all the entries in the time period (up to ec 01) and then copyedited in a second commit, pruning for notability.

Things to look out for:
Did I prune something important that should be restored?
Is there an ongoing project I can summarize? i.e. Each of the martial arts overhaul PRs are folded into one "overhauled all the martial arts" entry.

### CODE REVIEW

File: data/changelog.txt [hunk 1]
```diff
@@ -333,6 +363,20 @@ Added ability to toggle minimap on and off in Look Around window.
 Add visual indicators for dead zombies that can still revive.
 Add run and crouch colors for the player's symbol in ASCII.
 Display status for all long activities.
+Add option to increase search radius in map view.
+Enable autosave by default.
+CBM Auto Start thershold options.
```
Comment (I-am-Erk) [inline data/changelog.txt]:
```suggestion
CBM Auto Start threshold options.
```

File: data/changelog.txt [hunk 2]
```diff
@@ -385,6 +433,28 @@ Cbms harvested from NPC are filthy and faulty.
...
+Giant animals drop mutant meat, has negative effects if relied upon.
+Balanced bio repair nanobots.
+Standardized chemical powders for 1 unit = 1/100 mol.
+Apply item spawn scaling factor to monster drops.
+Balance bionic power use for realistic values.
+Allow large and huge creatures to move through underbrush.
+Mouse view was truncating last row of information.
+Draw debug vehicle autopilot AI and restore previous behaviour
+Fix accidental IR vision
+Revert #35148 from ifreund/player-to-character
```
Comment (I-am-Erk) [inline data/changelog.txt]: Do we need this listed?

Comment (kevingranade) [reply data/changelog.txt]: Nope, missed it somehow.

File: data/changelog.txt [hunk 3]
```diff
@@ -154,6 +154,22 @@ Allow martial arts techniques to target humanoids.
...
 Basecamps: add an emergency recall option.
+Allow shooting out lights in the lab.
+Add Auto start mode to power gen bionics.
+Introduces safe fuel mod and flags for bionics.
+Create template from already existing character.
+Allow extend to work for mutation_branch::category.
+Basecamp storage zone : to populate camp inventory.
+JSONize loot zones.
+Added an ability to geiger-scan NPCs.
+Dynamic NPC spawn anywhere on overmap - not just near player
```
Comment (kevingranade) [inline data/changelog.txt]:
```suggestion
Dynamic NPC spawn anywhere on overmap - not just near player.
```

File: data/changelog.txt [hunk 4]
```diff
@@ -154,6 +154,22 @@ Allow martial arts techniques to target humanoids.
...
+Spawn some chemicals with random charges
```
Comment (kevingranade) [inline data/changelog.txt]:
```suggestion
Spawn some chemicals with random charges.
```

**Pull request merged**